source/features/shared/footer/components/Footer.scss

```diff
@@ -99,8 +99,8 @@
         line-height: 1.6;
 
         span {
-          color: rgba(235, 34, 86, 0.5);
-          font-family: ProximaNova, sans-serif;
+          color: rgba(31, 193, 195, 0.3);
```

Comment:
@DeeJayElly could you change this approach to CSS styling? This way you can use SCSS variables, that will be used for the different network versions.

Reply:
Fixed